### DISCUSSION TITLE
doubt

### DISCUSSION THREAD

**Discussion by ninjadoc107** [Rapptz/discord.py#8428] [General]
how to use selfbot new version?
when I try to use a user token even using bot=False returns TypeError: Client.run() got an unexpected keyword argument 'bot'

**Comment by Bluesy1** [comment]
Selfbotting is against Discord's TOS and as such is not supported in Discord.py anymore

**Comment by Rapptz** [comment]
You can't. It got removed because it's against the terms of service.

**Reply by ninjadoc107** [reply]
old versions no longer work, and I didn't understand why they took it, since there was always TOS for selfbots..
